tests/stdlib/tparsecfg.nim

```diff
@@ -1,23 +1,76 @@
 discard """
-  output: '''OK'''
+  targets: "c js"
 """
 
 #bug #6046
```

Comment:
space: `#bug #6046` => `# bug #6046`, and also `# bug #6046` needs to be scoped inside a block otherwise it seems the whole file tests for this bug